README.md

```diff
@@ -12,3 +12,21 @@ page](https://docs.github.com/en/get-started/writing-on-github/working-with-adva
 might help with the notation for mathematical expressions.
 
 $T(n) \in O(f(n)) \iff \exists c, n_0: T(n) \leq c \cdot f(n) \forall n \geq n_0$
+
+# Response
+First, a translation of Big O out of mathematical notation into words, so I can understand it more clearly.
+A function $T(n)$ is an element of Big O of another function $f(n)$ if and only if there exists a value $c$ and $n_0$ where $T(n) \leq c(f(n))$ for all $n \geq n_0$.
+
+Consider the functions $g(n)=\log_{2}(n)$ and $f(n)=\log_{5}(n)$. Through the logarthim change of base property, the function $g(n)$ can be rewritten as $g(n)=\frac{\log_{5}(n)}{\log_{5}(2)}$. Note that the denominator in this function is a constant, which if we say $k = \frac{1}{\log{5}{2}}$ and substitute that in, we get the equation $g(n)=k\log_{5}(n)$
+
+From this equation, it is evident that $g(n)$ satisfies the definition of Big O such that $g(n) \in O(\log_{5}(n))$. In the same way, we can show that $\log_{5}(n) \in O(\log_{2}(n))$. Therefore, $O(\log_{2} n)$ and $O(\log_{5} n)$ are equivalent.
```

Comment:
This is what I was trying to say before. Do I need to show that $g(n)$ satisfies the definition of Big O, or is this sufficient?

Reply:
You cannot assume any particular $g(n)$ or $f(n)$ for this proof.